package.json

```diff
@@ -10,8 +10,9 @@
     "build": "ember build",
     "lint": "lint-all-the-things",
     "start": "ember server",
-    "test": "npm run lint && npm run test-addon",
-    "test-addon": "EMBER_TRY_SCENARIO=${EMBER_TRY_SCENARIO:=default} && ember try:one $EMBER_TRY_SCENARIO --- COVERAGE=true ember test"
+    "test": "npm run lint && npm run test-addon && npm run test-cli",
+    "test-addon": "EMBER_TRY_SCENARIO=${EMBER_TRY_SCENARIO:=default} && ember try:one $EMBER_TRY_SCENARIO --- COVERAGE=true ember test",
+    "test-cli": "_mocha --recursive tests/unit/lib/"
```

Comment:
https://github.com/gotwarlost/istanbul/issues/436

I changed it to mocha since we don't use istanbul.